### PR TITLE
Introduce QmlWeb variable, don't pollute global

### PR DESCRIPTION
**Pull request by ChALkeR**
Fixes: https://github.com/qmlweb/qmlweb/issues/51

We might want to add more methods/variables to the QmlWeb variable.
This does not deal with enums on a purpose, those should be covered by #271.

/cc @Plaristote @akreuzkamp @stephenmdangelo — PTAL.

### CODE REVIEW

File: src/qtcore/qmlstructure.js [hunk 1]
```diff
@@ -255,6 +255,9 @@ function loadParser() {
           throw new Error('Could not load QmlWeb parser!');
       }
       (new Function(xhr.responseText))();
+      // TODO: remove in 0.2, qmlweb.parser should set to QmlWeb if QmlWeb is defined
+      QmlWeb.qmlweb_parse = QmlWeb.qmlweb_parse || qmlweb_parse;
+      QmlWeb.qmlweb_jsparse = QmlWeb.qmlweb_jsparse || qmlweb_jsparse;
```
Comment (akreuzkamp) [inline src/qtcore/qmlstructure.js]: What do you think about renaming that to `QmlWeb.parse` and `QmlWeb.parseJs`? :)

**Comment by akreuzkamp** [comment]
lgtm :)

File: src/qtcore/qmlstructure.js (hunk 1 above)
Comment (ChALkeR) [inline src/qtcore/qmlstructure.js]: Will do, thanks! =)

**Comment by ChALkeR** [comment]
Merged.

Corresponding parser change: https://github.com/qmlweb/qmlweb-parser/commit/bb2aeaa45a8d69b95d3e4d85e3e1a7fd10d8381f

**Pull request merged**